research/query_service/ir/compiler/src/main/java/com/alibaba/graphscope/common/IrPlan.java

```diff
@@ -615,10 +615,14 @@ public byte[] toPhysicalBytes(Configs configs) throws BuildPhysicalException {
         return bytes;
     }
 
-    public String getPlanAsJson() {
+    public String getPlanAsJson() throws OpArgIllegalException {
         String json = "";
         if (ptrPlan != null) {
-            json = irCoreLib.printPlanAsJson(ptrPlan);
+            FfiError e = irCoreLib.printPlanAsJson(ptrPlan);
+            if (e.code != ResultCode.Success) {
+                throw new OpArgIllegalException(OpArgIllegalException.Cause.INVALID_TYPE, e.msg);
```

Comment:
Cannot find a proper `Exception` to throw.

Reply:
InterOpIllegalArgException is used to handle error from ir_core, I have updated